### PR TITLE
Wip/hughsie/kenv

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupdplugin/fu-common.c [hunk 1]
```diff
@@ -3282,6 +3286,44 @@ fu_common_align_up (gsize value, guint8 alignment)
 	return value_new;
 }
 
+/**
+ * fu_common_kenv:
```
Comment (superm1) [inline libfwupdplugin/fu-common.c:3290]: How about fu_common_bsd_kenv to make it clearer to future selfs this is only intended for use with bsd?

**Review by superm1 — DISMISSED** [review]

File: libfwupdplugin/fu-common.c (hunk 1 above)
Comment (hughsie) [reply libfwupdplugin/fu-common.c:3290]: is kenv used on macos too? that's the bit i wasn't sure of.

Comment (hughsie) [reply libfwupdplugin/fu-common.c:3290]: Yes, this seems used on OS-X too. Maybe a better approach would be to have a libfwupdplugin/fu-kenv.[c|h] with a symbol like fu_kenv_get_string()? That might be better for the long term.

Comment (superm1) [reply libfwupdplugin/fu-common.c:3290]: sounds good

Comment (hughsie) [reply libfwupdplugin/fu-common.c]: Done.

**Review by superm1 — APPROVED** [review]

**Pull request merged**